app/assets/stylesheets/custom.css.scss

```diff
@@ -57,6 +57,16 @@ input[type="submit"] {
   box-shadow: none;
 }
 
+#small_submit_button {
+  width: 20%;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered -moz-box-shadow, -webkit-box-shadow, box-shadow, display, text-align, width